tests/dictzip_test.py

```diff
@@ -1,11 +1,14 @@
 import gzip
 import tempfile
+from functools import partialmethod
 from pathlib import Path
+from typing import Callable
 
-import idzip as _idzip  # noqa: F401
 from glossary_errors_test import TestGlossaryErrorsBase
 
-from pyglossary.os_utils import runDictzip
+from pyglossary.os_utils import _dictzip, _idzip
```

Comment:
If you want to test them, perhaps shouldn't make them private.

Reply:
It is preferred to make a «granular» test, meanwhile not to «export» functions for user, since Python allows it. Indirect test for «private» functions needs another few lines of code.

If you wish, let us do one of:
- Strip an underscore in names of functions
- Test its indirect with a `runDictzip` optional arg
- (Or keep things as is)

Reply:
Higher level functions are preferred for testing. Let's just test runDictzip.

Reply:
Yes. Also remove nozip please. And don't need to loop over 2 functions.

Reply:
And unmark as draft.